linux_os/guide/system/accounts/accounts-session/accounts_tmout/oval/shared.xml

```diff
@@ -37,8 +37,10 @@
     {{% if filepath %}}
     <ind:filepath>{{{ filepath }}}</ind:filepath>
     {{% endif %}}
-    {{% if product in ['sle12', 'sle15'] or "ubuntu" in product %}}
+    {{% if product in ['sle12', 'sle15'] or product in ['ubuntu1804', 'ubuntu2204'] %}}
```

Comment:
we should not split this rule
I belive the solution as is now is way better than what is in DISA's stig document.

Reply:
Would you recommend to drop the PR if we follow CIS? The original remediation seems to work well so not sure if this PR is even needed.

Reply:
I believe you will still want to add a ubuntu specific ansible, as the shared ansible does not cover our case, see bash remediation. But for the oval you could drop the changes.